db/migrate/20220309052057_add_client_bypass_note_to_verification_attempt.rb

```diff
@@ -0,0 +1,5 @@
+class AddClientBypassNoteToVerificationAttempt < ActiveRecord::Migration[6.1]
+  def change
+    add_column :verification_attempts, :client_bypass_request, :text
```

Comment:
so this is adding a new column to the verification attempts table correct?

Reply:
correct!